image/src/mobilenet/teachable-mobilenet.ts

```diff
@@ -22,38 +22,55 @@ import { TensorContainer } from '@tensorflow/tfjs-core/dist/tensor_types';
 import { CustomCallbackArgs, equalStrict } from '@tensorflow/tfjs';
 import { CustomMobileNet, Metadata, loadTruncatedMobileNet, ClassifierInputSource } from './custom-mobilenet';
 
+import * as seedrandom from 'seedrandom';
+
+const VALIDATION_FRACTION = 0.15;
 
 export interface TrainingParameters {
     denseUnits: number;
     epochs: number;
     learningRate: number;
+    batchSizeFraction: number;
+}
+
+interface Sample {
+    data: Float32Array;
+    class: number[];
```

Comment:
`class` is a reserved word in JavaScript. Lets change this key so we don't create odd situations like with destructuring:

`const { class } = sample;`

Reply:
great catch, you're totally right.